tools/merge_chainspec_accounts.js

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env node
+// Called like this:
+// node merge_chainspec_accounts.js chainspec.json accounts.json
```

Comment:
you should also explain where chainspec and accounts comes from.

Reply:
👍 
I'll add some docs for that script.